### PR TITLE
Update dibs.txt

### PR DESCRIPTION
**Pull request by davidteather**

### CODE REVIEW

**Comment by AdamSchmidty** [comment]
your code is trash kid

File: testFolder/dibs.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 hi!
+Adam Schimdt
```
Comment (davidteather) [inline testFolder/dibs.txt:2]: LGTM!!!!

**Review by davidteather — COMMENTED** [review]
LGTM!!!!!

**Comment by stevenlai1688** [comment]
please revert

File: testFolder/dibs.txt (hunk 1 above)
Comment (AdamSchmidty) [reply testFolder/dibs.txt:2]: LGTM!!!

**Comment by AdamSchmidty** [comment]
no

**Pull request merged**